### DISCUSSION TITLE
Problem with paramcols in iMubu help file

### DISCUSSION THREAD

**Discussion by mattyostrowski** [Ircam-RnD/catart-mubu#114] [Bug, Issues, and Suggestions]
If you go into  "mapping marker attributes to matrix columns" in Views-1 of the iMubu helpfile, there's a message box "view 1 paramcol duration time" with the comment "set given parameter column by "special" keyword "time"". Clicking on this displays the  label column (which I guess makes sense, as there is no Duration column) and the contents of col1, rather than the time. 

If I change the message to 'paramcols' plural, since there's more than one col referenced , then it displays the label track only.

I don't know if this is just a problem with the helpfile or iMubu....

**Comment by RiccardoBorghesi** [comment]
A special ("fake") index is a keyword which says that the parameter in question does not correspond to a column of the matrix. "view 1 paramcol duration time" means that the displayed duration will be equal to the difference between the timetag of the next marker and the timetag of the current marker. This feature actually deserves to be explained precisely, which is done in the new version of the imubu help, which we will send you soon.

**Reply by mattyostrowski** [reply]
Just catching up with this -- I'd maybe suggest that you put simultaneous views marker/matrix in the Mapping subpatcher -- makes a lot more sense if you can see both data representations at once. Also, at the bottom, I think you should be able to switch between views to compare.  I'd also double-check them to make sure they're working as advertised -- on the bottom right if 'duration' is the difference between the timetags, I'd expect to see '6 6 6 6 6', but I'm not. Also, I'm not quite sure what the 'label' messdage does here. The imubu on the bottom right doesn't have a label column, and if I hook it up to the main one, which does, I don't see a label appearing. And you show the 'fake' message 'markerindex', but the examples use 'index'.
